src/clj/game/cards/events.clj

```diff
@@ -2023,8 +2022,7 @@
      nil
      nil
      (effect (register-events {:pre-rez-cost {:req (req (ice? target))
-                                              :msg (msg "add an additional cost of " (:cost target)
-                                                        " [Credits] to rez " (card-str state target))
+                                              :msg (msg "add an additional cost (currently unknown) to rez " (card-str state target))
```

Comment:
Why is this "currently unknown"?

Reply:
If the ice is facedown, the runner doesn't know how much it costs. X is only determined when the corp goes to rez the ice. The runner might make different decisions if they know the card will cost 2 to rez vs 10. The phrasing might could be changed, but we shouldn't be writing the actual cost of the facedown ice in the log either way.

Reply:
Maybe "to double the rez cost of X"?

Reply:
I like that!